code/obj/machinery/door/airlock.dm

```diff
@@ -219,6 +219,19 @@ Airlock index -> wire color are { 9, 4, 6, 7, 5, 8, 1, 2, 3 }.
 		. = ..()
 		STOP_TRACKING
 
+	get_desc()
+		var/healthpercent = src.health/src.health_max * 100
+		switch(healthpercent)
+			if(90 to 99) //dont want to clog up the description unless it's actually damaged
+				. += "It seems to be in mostly good condition"
+			if(75 to 89)
+				. += "[src] seems slightly [pick("dinged up", "dented", "damaged", "scratched")]"
+			if(50 to 74)
+				. += "[src] looks [pick("busted", "damaged", "messed up", "dented")]."
+			if(25 to 49)
+				. += "[src] looks [pick("quite", "pretty", "rather", "notably")] [pick("mangled", "busted", "messed up", "wrecked", "destroyed", "haggard")]."
```

Comment:
Seems odd to make the description randomize on each examine, I can't think of anything else that does this.

Reply:
Idea is from the drone's health on description indicators, which chooses a random synonym for flavor's sake.